### PR TITLE
Account for xScale when editing layer timings

### PR DESCRIPTION
**Pull request by stianjensen**
Editing length of layers after #133 did not work as expected

### CODE REVIEW

**Comment by sigvef** [comment]
We should probably round all these values to whole numbers, as nin doesn't work with layers that start/stop on fractional frame numbers.

**Comment by stianjensen** [comment]
ping

File: nin/frontend/app/scripts/controllers/bottom.js [hunk 1]
```diff
@@ -28,17 +28,17 @@
       };
 
       $scope.dragResizeLayer = function(event, ui, layer) {
-        if (ui.position.left != layer.startFrame) {
+        if (ui.position.left != layer.startFrame * $scope.xScale) {
```
Comment (sigvef) [inline nin/frontend/app/scripts/controllers/bottom.js]: You probably also need to round to int here as well.

**Comment by stianjensen** [comment]
ping

**Pull request merged**